src/app/containers/README.md

```diff
@@ -1,36 +0,0 @@
-# Containers
-
-## Overview
```

Comment:
```suggestion
We are now creating containers that are agnostic to data source within [Psammead] (https://github.com/bbc/psammead), our component library. We are in the process of migrating all existing reusable containers out of Simorgh and into Psammead. If you need to add a container to be used in this application, please first open an issue to discuss where it should best be placed, within Psammead or Simorgh. 
```

Reply:
Will remove them here then have another PR to add a very simple readme to the base of every major dir

Reply:
https://github.com/bbc/simorgh/issues/2873